packages/components/src/__future__/Tag/Tag/_docs/MigrationGuide.mdx

```diff
@@ -0,0 +1,168 @@
+
+# Tag migration guide
+
+A breif guide on how and why to migrate from Kaizen's legacy `Tag` to the `future` release.
+
+## Why the change?
+
+Tag was rebuilt to allow for greater composability as the number of cases for a destinct Tags could not be solved through the use of `variant` with impacting the developer experience for existing user. The flexibility of tag content as `children` also addresses several accessibility issues that the orginal could not solve intuatively.
+
+## Component and API changes at a glance
+
+There were a number key changes between legacy and the current `Tag` to achieve the flexbility we were aiming to provide, we've listed these below but also go into futre detail in the [migration examples](/docs/components-tag-future-migration-guide--migrationguide#migration-examples).
+
+- `variant` prop has been removed
+  - This has been superseded by a combination of the `color` and `icon` prop
+- `inline` prop and the default `margin` on all Tags has been removed.
+  - This can be directly controlled through `classNameOverride` or a `flex-gap` on the container.
+- `size` has been removed and all tags are the same size.
+- `truncationWidth` has been removed as child content should be controlled by the consumer.
+- `dismissable` prop has been removed.
+  - A `DismissableTag` component is now available in its place.
+- `Avatar` and all child content must now be composed by the consumer.
+  - This offers a greater degree of flexibility to ensure that the right accessible context can be provided to the end user.
+
+
+## Migration examples
+
+The following are example of migrating from previously available props to the composable solution.
+
+### Status
+
+With the `variant` prop there are four value ascoiated with statuses in the legacy tag:  `statusLive`, `statusDraft`, `statusClosed`, `statusAction`.
+
+For the most part these can all be replaced easily with the use of the `color` prop, with exception to the `live` status.
+
+#### Before
+
+<Canvas of={LegacyTagStories.Status} />
+
+
+#### After
+
+<Canvas of={TagMigrationStories.StatusMigration} sourceState="shown"/>
+
+While the `LiveIcon` is exported from `@kaizen/components` the legacy `Tag` used an animated implementation, which is not currently available. As an interim solution you can use the following story's code snippet to create a `LiveIconComponent`. This can then be passed into as `Children` of the `Tag`.
+
+<Canvas of={TagMigrationStories.LiveIconComponentStory} sourceState="hidden"/>
+
+### Validation
+
+Following the same pattern as status, `validationPositive`, `validationInformative`, `validationNegative`, `validationCautionary` can all be replace with the `color` and `icon` prop.
+
+Note: because the icons convey meaning, they should have an accessible label that provides context to the tag. A contextual `aria-label`and role of `img` addresses one of the [accessibility issues](https://github.com/cultureamp/kaizen-discourse/issues/102) with the legacy Tag.
```

Comment:
```suggestion
Note: Because the icons convey meaning, they should have an accessible label that provides context to the tag. A contextual `aria-label` and role of `img` addresses one of the [accessibility issues](https://github.com/cultureamp/kaizen-discourse/issues/102) with the legacy Tag.
```

Rather than link to the a11y issue, I'd probably just provide an example snippet with the correct expected code.